Twitter.py

```diff
@@ -0,0 +1,27 @@
+import tweepy
+
+
+
+class Twitter(object):
+    @staticmethod
+    def home_timeline(account, count):
+        auth = tweepy.OAuthHandler(TWITTER_CONSUMER_KEY, TWITTER_CONSUMER_SECRET)
+        auth.set_access_token(TWITTER_ACCESS_TOKEN,
+                              TWITTER_ACCESS_TOKEN_SECRET)
+
+        api = tweepy.API(auth, wait_on_rate_limit=True)
+        return api.user_timeline(screen_name=account, count=count)
+
+
+class Tweet(object):
+    def __init__(self, id, screen_name, text, retweeted, favorite_count, retweet_count, created_at):
+        self.id = id
+        self.screen_name = screen_name
+        self.text = text
+        self.retweeted = retweeted
+        self.favorite_count = favorite_count
+        self.retweet_count = retweet_count
+        self.created_at = created_at
+
+    def __repr__(self):
+        return '<Tweet id=%s, screen_name=%s>' % (self.id, self.screen_name)
```

Comment:
Would probably use https://marshmallow.readthedocs.io to represent JSON responses.